frontend/__test__/SelectTest.test.js

```diff
@@ -0,0 +1,55 @@
+import { renderRouter, screen, fireEvent } from 'expo-router/testing-library';
```

Comment:
kan filen ikke bare hedde Select.test.js (og så er det stilen for vores testfiler)